src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PackageRestore/PackageRestoreDataSource.cs

```diff
@@ -148,30 +145,23 @@ private async Task<bool> RestoreCoreAsync(ProjectRestoreInfo restoreInfo)
             // Prevent overlap until Restore completes
             bool success = await joinableTask;
 
-            await HintProjectDependentFileAsync(restoreInfo);
+            HintProjectDependentFile(restoreInfo);
 
             return success;
         }
 
-        private Task HintProjectDependentFileAsync(ProjectRestoreInfo restoreInfo)
+        private void HintProjectDependentFile(ProjectRestoreInfo restoreInfo)
         {
-            // Hint to CPS that the assets file "might" have changed and therefore
```

Comment:
That block of comments still makes sense after this change.

Reply:
I just moved it lower down.